### PR TITLE
chore(PaparaTopluluk): update logo url

### PR DESCRIPTION
**Pull request by eggsy**
## Description 
They updated the logo. _Not sure if this is the correct way to update the assets._

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

### CODE REVIEW

File: websites/P/Papara Topluluk/metadata.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	},
 	"url": "topluluk.papara.com",
 	"version": "1.0.7",
-	"logo": "https://cdn.rcd.gg/PreMiD/websites/P/Papara%20Topluluk/assets/logo.png",
+	"logo": "https://i.imgur.com/3L9gPyD.png",
```
Comment (theusaf) [inline websites/P/Papara Topluluk/metadata.json:14]: Image should be 512x512px

**Review by theusaf — CHANGES_REQUESTED** [review]
Additionally, ensure to resolve the automated checks

File: websites/P/Papara Topluluk/metadata.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
 	},
 	"url": "topluluk.papara.com",
 	"version": "1.0.7",
```
Comment (DarkVillager) [inline websites/P/Papara Topluluk/metadata.json:13]:
```suggestion
	"version": "1.0.8",
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

**Review by DarkVillager — APPROVED** [review]
LGTM Boss

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**